biomage/experiment/experiment.py

```diff
@@ -0,0 +1,154 @@
+import itertools
+import copy
+from pprint import pprint
+
+import click
+import boto3
+from botocore.exceptions import ClientError
+from deepdiff import DeepDiff
+
+
+@click.group()
+def experiment():
```

Comment:
you should not forget to update the README file with instructions about how to use this